services/net/contentmigration/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM mcr.microsoft.com/dotnet/sdk:9.0 as build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 as build
```

Comment:
I forgot to roll these back.  Locally they build fine, but in Openshift they do not.